apps/engineering/content/design/components/date-time.example.tsx

```diff
@@ -0,0 +1,39 @@
+"use client";
+import { RenderComponentWithSnippet } from "@/app/components/render";
+import { Row } from "@/app/components/row";
+import { DateTime, type Range } from "@unkey/ui";
+import { useState } from "react";
+
+export const DateTimeExample: React.FC = () => {
+  const [date, setDate] = useState<Range>();
+  const [startTime, setStartTime] = useState<string>("09:00");
+  const [endTime, setEndTime] = useState<string>("17:00");
+
+  const handleChange = () => {
```

Comment:
I believe this will receive an actual modified time in the future?

`const handleChange = (dateTime: SomeSaneType) => {}`